### PR TITLE
Less qt2

### PR DESCRIPTION
**Pull request by SylvestreG**
Remove some include and QFiles.

### CODE REVIEW

File: core/inc/com/centreon/broker/misc/misc.hh [hunk 1]
```diff
@@ -29,6 +29,25 @@ namespace misc {
   std::string temp_path();
   std::list<std::string> split(std::string const& str, char sep);
   uint16_t crc16_ccitt(char const* data, unsigned int data_len);
+
+  class Uuid {
```
Comment (bouda1) [inline core/inc/com/centreon/broker/misc/misc.hh]: Would be better to move it into misc/uuid.{cc,hh}

File: core/src/extcmd/command_request.cc [hunk 2]
```diff
@@ -17,7 +17,7 @@
 */
 
 #include <cstdlib>
-#include <QUuid>
+#include "com/centreon/broker/misc/misc.hh"
```
Comment (bouda1) [inline core/src/extcmd/command_request.cc]: Then we would have com/centreon/broker/misc/uuid.hh which is more explicit.

File: core/src/extcmd/command_request.cc [hunk 3]
```diff
@@ -36,8 +36,10 @@ using namespace com::centreon::broker::extcmd;
  *  Default constructor.
  */
 command_request::command_request()
-  : uuid(QUuid::createUuid().toString().toStdString()),
-    with_partial_result(false) {}
+  : with_partial_result(false) {
+  misc::Uuid tmp_uuid;
+  uuid = tmp_uuid.to_string();
+}
```
Comment (bouda1) [inline core/src/extcmd/command_request.cc]: Itn't it possible to call directly something like uuid{misc::Uuid().to_string()} before the {} block?

File: core/src/file/qt_fs_browser.cc [hunk 4]
```diff
@@ -16,8 +16,9 @@
 ** For more information : contact@centreon.com
 */
 
-#include <QDir>
-#include <QFile>
+#include <sys/types.h>
+#include <regex.h>
```
Comment (bouda1) [inline core/src/file/qt_fs_browser.cc]: Should not we remove this?

File: core/src/file/qt_fs_browser.cc [hunk 5]
```diff
@@ -40,21 +41,35 @@ qt_fs_browser::~qt_fs_browser() {}
  *
  *  @return List of elements within the directory.
  */
+#include <iostream>
 fs_browser::entry_list qt_fs_browser::read_directory(
```
Comment (bouda1) [inline core/src/file/qt_fs_browser.cc]: We should change this name now :-) : qt_fs_browser...

File: core/src/misc/misc.cc [hunk 6]
```diff
@@ -63,4 +64,59 @@ uint16_t misc::crc16_ccitt(char const* data, unsigned int data_len) {
     crc = ((crc >> 4) & 0x0fff) ^ crc_tbl[((crc ^ c) & 15)];
   }
   return ~crc & 0xffff;
+}
+
+misc::Uuid::Uuid() {
```
Comment (bouda1) [inline core/src/misc/misc.cc]: idem here

File: core/inc/com/centreon/broker/misc/misc.hh (hunk 1 above)
Comment (SylvestreG) [reply core/inc/com/centreon/broker/misc/misc.hh]: done

File: core/src/extcmd/command_request.cc (hunk 2 above)
Comment (SylvestreG) [reply core/src/extcmd/command_request.cc]: done

File: core/src/extcmd/command_request.cc (hunk 3 above)
Comment (SylvestreG) [reply core/src/extcmd/command_request.cc]: done

File: core/src/file/qt_fs_browser.cc (hunk 4 above)
Comment (SylvestreG) [reply core/src/file/qt_fs_browser.cc]: not yet regcomp is used in qt_fs_browser.

File: core/src/file/qt_fs_browser.cc (hunk 5 above)
Comment (SylvestreG) [reply core/src/file/qt_fs_browser.cc]: done stl_fs_browser !

File: core/src/misc/misc.cc (hunk 6 above)
Comment (SylvestreG) [reply core/src/misc/misc.cc]: done

File: core/src/processing/feeder.cc [hunk 7]
```diff
@@ -60,7 +58,9 @@ feeder::feeder(
 /**
  *  Destructor.
  */
-feeder::~feeder() {}
+feeder::~feeder() {
+  this->bthread::~bthread();
```
Comment (bouda1) [inline core/src/processing/feeder.cc]: Does this call really necessary? You set bthread destructor as virtual, so the work should be done...

**Pull request merged**